### PR TITLE
Choose File button UI improvement

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Fixes: #433

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

editor3

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by VladimirMikulic** [comment]
@NitinBhasneria 'improved' is too generic. You need to provide **what** was improved and the most important - **why** it was improved. Your commit message should include this as well as PR description.

**Comment by NitinBhasneria** [comment]
> @NitinBhasneria 'improved' is too generic. You need to provide **what** was improved and the most important - **why** it was improved. Your commit message should include this as well as PR description.

OK thanks @VladimirMikulic will remember next time.

**Comment by shreyaa-sharmaa** [comment]
Now that #472 is merged, I think this should be merged too. This would create consistency in button styles.
cc: @jywarren @emilyashley @VladimirMikulic @keshav234156 @Shulammite-Aso

**Comment by NitinBhasneria** [comment]
Yes, @shreyaa-sharmaa also File name error is needs to be removed before this. ##461

File: src/modules/PublicLab.MainImageModule.js [hunk 1]
```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
 
     });
 
+    var input = document.getElementById( 'thumbnail-img' );
+    var infoArea = document.getElementById( 'thumbnail-filename' );
+
+    input.addEventListener( 'change', showFileName );
+
+    function showFileName( event ) {
+
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:157]: Redundant blank line.

File: src/modules/PublicLab.MainImageModule.js [hunk 2]
```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
 
     });
 
+    var input = document.getElementById( 'thumbnail-img' );
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:151]: Redundant spaces around selector.

File: src/modules/PublicLab.MainImageModule.js [hunk 3]
```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+    var infoArea = document.getElementById( 'thumbnail-filename' );
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:152]: Here too.

File: src/modules/PublicLab.MainImageModule.js [hunk 4]
```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+
+    input.addEventListener( 'change', showFileName );
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:154]: As well as here.

File: src/modules/PublicLab.MainImageModule.js [hunk 5]
```diff
@@ -148,6 +148,18 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
...
+
+    function showFileName( event ) {
```
Comment (VladimirMikulic) [inline src/modules/PublicLab.MainImageModule.js:156]: The style should be consistent with the rest of the file.

Comment (NitinBhasneria) [reply src/modules/PublicLab.MainImageModule.js:156]: Thanks for the review @VladimirMikulic. Solved.

**Comment by cesswairimu** [comment]
@VladimirMikulic kindly do a final review against your changeswhen you can. Thanks

**Comment by NitinBhasneria** [comment]
Some CSS is updated.
Thanks. 
mainimage

This is updated gif.

File: dist/PublicLab.Editor.css [hunk 6]
```diff
@@ -2,6 +2,42 @@
...
+#thumbnail-filename {
+    margin: 0px 0px 5px 4px;
+}
+
+.thumbnailBtn:hover {
+  background-color: #efefef;
+}
+
+.thumbnailBtn:focus { 
+  outline:none
```
Comment (VladimirMikulic) [inline dist/PublicLab.Editor.css:14]: Space needed. Keep a consistent style. Use a tool.

File: src/modules/PublicLab.MainImageModule.js (hunk 5 above)
Comment (VladimirMikulic) [reply src/modules/PublicLab.MainImageModule.js:153]: I don't see it solved?

**Comment by NitinBhasneria** [comment]
Sorry for that, I have added some commits. Thanks

**Comment by cesswairimu** [comment]
Great thanks @NitinBhasneria looking great :tada:  and thanks @VladimirMikulic  for the reviews :heart:

**Pull request merged**